pkg/api/persistentvolume/util.go

```diff
@@ -40,11 +40,11 @@ func VisitPVSecretNames(pv *api.PersistentVolume, visitor Visitor) bool {
 	switch {
 	case source.AzureFile != nil:
 		if source.AzureFile.SecretNamespace != nil && len(*source.AzureFile.SecretNamespace) > 0 {
-			if len(source.AzureFile.SecretName) > 0 && !visitor(*source.AzureFile.SecretNamespace, source.AzureFile.SecretName) {
+			if len(source.AzureFile.SecretName) > 0 && !visitor(*source.AzureFile.SecretNamespace, source.AzureFile.SecretName, true) {
```

Comment:
nit: when it is unclear what the value means, it's nice to have a comment, for example:

```go
!visitor(
    *source.AzureFile.SecretNamespace,
    source.AzureFile.SecretName,
    true /* kubeletVisible */) {
```

But it's a personal preference, up to you if you want to do that.

Reply:
agree, it is more next developer friendly :)